src/Github/GithubHookProcessor.php

```diff
@@ -32,47 +32,43 @@ public function __construct(GithubClient $client)
      *
      * Github events: issue_comment, pull_request_review_comment
      */
-    public function processPendingAuthorLabel(Event $event, array $payload): void
+    public function processPendingAuthorLabel(Payload $payload): void
     {
-        if (!\in_array($payload['action'], ['created', 'synchronize'], true)) {
+        if (!\in_array($payload->action(), ['created', 'synchronize'], true)) {
             return;
         }
 
-        $issueKey = 'issue_comment' === $event->toString() ? 'issue' : 'pull_request';
+        if ($payload->commentAuthorId() !== $payload->issueAuthorId()) {
```

Comment:
Let's introduce a method that does just that instead of exposing these properties?

Reply:
Will do, but will change it step by step

Reply:
commentAuthorEqualsIssueAuthor() ?

Other naming proposals @greg0ire ?

Reply:
`isTheCommentFromTheAuthor`?

Reply:
Love it, thanks

Reply:
`isTheCommentFromTheAuthor`

Exactly the kind of high-level naming I was hoping for :)
That will make the code easier to understand 👍